iam/policy-templates/ci-cd.json

```diff
@@ -102,9 +102,7 @@
         "lambda:*",
         "es:*",
         "sns:*",
-        "states:StartExecution",
-        "states:DescribeExecution",
-        "states:CreateStateMachine"
+        "states:*"
```

Comment:
you are granting wider set of permissions than needed

Reply:
These apply only specifically to the dss-* state machines, so I don't think it matters.